src/hooks/useSubmitAuction.tsx

```diff
@@ -0,0 +1,207 @@
+    const minimumBiddingAmountPerOrder = utils.parseUnits(
+      minBuyAmountPerOrder,
+      biddingTokenDecimals
+    )
+    const minFundingThresholdInAtoms = utils.parseUnits(minFundingThreshold, biddingTokenDecimals)
+    const sellAmountInAtoms = utils.parseUnits(sellAmount, auctioningTokenDecimals)
+
+    if (moment(auctionEndDate).isBefore(moment())) {
+      setError('auctionEndDate', {
+        type: 'manual',
+        message: 'Auction End Date should be in the future',
+      })
+
+      formHasErrors = true
+    }
+
+    if (
+      moment(orderCancellationEndDate).isBefore(moment()) ||
+      moment(orderCancellationEndDate).isAfter(moment(auctionEndDate))
+    ) {
+      setError('orderCancellationEndDate', {
+        type: 'manual',
+        message: 'Order cancellation End Date should be in the future and before auction End Date',
+      })
+
+      formHasErrors = true
+    }
+
+    if (allowListManager && ADDRESS_REGEX.test(allowListManager)) {
+      const allowListManagerIsContract = await checkIsContract(sdk, allowListManager)
+      if (!allowListManagerIsContract) {
+        setError('allowListManager', {
+          type: 'manual',
+          message: `allowListManager should be a contract deployed in ${safe.network}`,
+        })
+        formHasErrors = true
+      }
+    } else {
+      // eslint-disable-next-line no-console
+      console.log('AllowListManager not set or not an address')
+      useDefaultAllowListManager = true
+    }
+
+    if (allowListData && ADDRESS_REGEX.test(allowListData)) {
+      const allowListDataIsContract = await checkIsContract(sdk, allowListData)
+      if (allowListDataIsContract) {
+        setError('allowListData', { type: 'manual', message: 'allowListData should be an EOA' })
+        formHasErrors = true
+      }
+    } else {
+      // eslint-disable-next-line no-console
+      console.log('AllowListData/Signer not set or not an address')
+      useDefaultAllowListData = true
+    }
+
+    if (formHasErrors) {
+      throw new Error('Form completed with errors')
+    }
+
+    const txs: Transaction[] = []
+
+    if (sellAmountInAtoms.gt(allowance)) {
+      txs.push({
+        to: auctioningToken.address,
+        value: '0',
+        data: auctioningToken.interface.encodeFunctionData('approve', [
+          easyAuction.address,
+          sellAmountInAtoms,
+        ]),
+      })
+    }
+
+    const valuesToSend: ValuesToSend = [
+      auctioningToken.address,
+      biddingToken.address,
+      moment(orderCancellationEndDate).unix().toString(),
+      moment(auctionEndDate).unix().toString(),
+      sellAmountInAtoms,
+      minBuyAmountInAtoms,
+      minimumBiddingAmountPerOrder,
+      minFundingThresholdInAtoms,
+      !!isAtomicClosureAllowed,
+      useDefaultAllowListManager ? '0x0000000000000000000000000000000000000000' : allowListManager,
+      useDefaultAllowListData
+        ? formatBytes32String('0x')
```

Comment:
formatBytes32String('0x')

is returning for me: 
```
"0x3078000000000000000000000000000000000000000000000000000000000000"
```

Sending over just '0x' should be fine. It does not need to be bytes32, if we don't have an allowlist manager contract specified